mods/ts/rules/gdi-structures.yaml

```diff
@@ -488,37 +502,52 @@ GAPLUG:
 	Power@hunterseeker:
 		RequiresCondition: (plug.hunterseekera || plug.hunterseekerb)
 		Amount: -50
+	Power@droppod:
+		RequiresCondition: (plug.droppoda || plug.droppodb)
+		Amount: -20
 	Pluggable@pluga:
 		Offset: 0,2
 		Conditions:
 			plug.ioncannon: plug.ioncannona
 			plug.hunterseeker: plug.hunterseekera
+			plug.droppod: plug.droppoda
 		Requirements:
-			plug.ioncannon: !build-incomplete && !plug.ioncannonb && !plug.ioncannona && !plug.hunterseekera
-			plug.hunterseeker: !build-incomplete && !plug.hunterseekerb && !plug.ioncannona && !plug.hunterseekera
+			plug.ioncannon: !plug.ioncannonb && !plug.ioncannona && !plug.hunterseekera && !plug.droppoda
+			plug.hunterseeker: !plug.hunterseekerb && !plug.ioncannona && !plug.hunterseekera && !plug.droppoda
+			plug.droppod: !plug.droppodb && !plug.ioncannona && !plug.hunterseekera && !plug.droppoda
 	WithIdleOverlay@ioncannona:
-		RequiresCondition: !build-incomplete && plug.ioncannona
+		RequiresCondition: plug.ioncannona
 		PauseOnCondition: disabled
 		Sequence: idle-ioncannona
 	WithIdleOverlay@hunterseekera:
-		RequiresCondition: !build-incomplete && plug.hunterseekera
+		RequiresCondition: plug.hunterseekera
 		PauseOnCondition: disabled
 		Sequence: idle-hunterseekera
+	WithIdleOverlay@droppoda:
+		RequiresCondition: plug.droppoda
+		PauseOnCondition: disabled
+		Sequence: idle-droppoda
 	Pluggable@plugb:
 		Offset: 1,2
 		Conditions:
 			plug.ioncannon: plug.ioncannonb
 			plug.hunterseeker: plug.hunterseekerb
+			plug.droppod: plug.droppodb
 		Requirements:
-			plug.ioncannon: !build-incomplete && !plug.ioncannona && !plug.ioncannonb && !plug.hunterseekerb
-			plug.hunterseeker: !build-incomplete && !plug.hunterseekera && !plug.ioncannonb && !plug.hunterseekerb
+			plug.ioncannon: !plug.ioncannona && !plug.ioncannonb && !plug.hunterseekerb && !plug.droppodb
+			plug.hunterseeker: !plug.hunterseekera && !plug.ioncannonb && !plug.hunterseekerb  && !plug.droppodb
+			plug.droppod: !plug.droppoda && !plug.ioncannonb && !plug.hunterseekerb  && !plug.droppodb
 	WithIdleOverlay@ioncannonb:
-		RequiresCondition: !build-incomplete && plug.ioncannonb
+		RequiresCondition: plug.ioncannonb
 		PauseOnCondition: disabled
 		Sequence: idle-ioncannonb
 	WithIdleOverlay@hunterseekerb:
-		RequiresCondition: !build-incomplete && plug.hunterseekerb
+		RequiresCondition: plug.hunterseekerb
 		PauseOnCondition: disabled
 		Sequence: idle-hunterseekerb
+	WithIdleOverlay@droppodb:
+		RequiresCondition: plug.droppodb
+		PauseOnCondition: disabled
+		Sequence: idle-droppodb
 	ProvidesPrerequisite@buildingname:
-	SelectionDecorations:
+	SelectionDecorations:
```

Comment:
Missing a newline at the end. See https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file?fbclid=IwAR1iuN8Luca-Y0FJh_CdNRt2eo5zTAHnSg4-SxFlNqywrT1shu3NHV0lgtc